### DISCUSSION TITLE
Update to Windows.AppSDK 1.6 status (release last Friday Sep 9th)

### DISCUSSION THREAD

**Discussion by matthewacme** [Live-Charts/LiveCharts2#1590] [Q&A]
anyone working on an update to the code to port over to use the Windows.AppSDK 1.6 release. It's majorly broken when attempting to run the LiveCharts in an WinUI app that has been ported over to the Windows.AppSDK 1.6 (from 1.5)

**Comment by beto-rodriguez** [comment]
Hi, see #1580
